internal/models/helmcharts/method_get.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package helmchartorgmodel
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChartGetResponse Response from getting a Chart.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.organization.fluxcd.helm.repository.chartmetadata.chart.GetChartResponse
+type VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChartGetResponse struct {
+
+	// Chart returned.
+	Chart *VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChart `json:"chart,omitempty"`
```

Comment:
HelmRepositoryChart? Is it repository or chart?

Reply:
This is chart not repository as you can see in swagger address it is like this
`vmware.tanzu.manage.v1alpha1.organization.fluxcd.helm.repository.chartmetadata.chart`
that's why its name is `VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChart `
do you want me to change it ?